src/orchestrator/unary.rs

```diff
@@ -140,7 +146,24 @@ pub async fn input_detection_task(
     let text_with_offsets = apply_masks(input_text, masks);
     let chunker_ids = get_chunker_ids(ctx, detectors)?;
     let chunks = chunk_task(ctx, chunker_ids, text_with_offsets).await?;
-    let detections = detection_task(ctx, detectors, chunks).await?;
+    let mut empty_chunk = HashMap::new();
+    empty_chunk.insert(
+        "en_regex".to_string(),
+        vec![Chunk {
+            offset: 0,
+            text: "".to_string(),
+        }],
+    );
+    let chunk_lenght = match chunks.get("en_regex") {
+        Some(chunk) => chunk.len(),
+        None => (0 - 1) as usize,
+    };
+    let returned_chunks = if chunk_lenght == 0 {
+        empty_chunk
+    } else {
+        chunks
+    };
+    let detections = detection_task(ctx, detectors, returned_chunks).await?;
```

Comment:
Can you update the PR to explain this? Also, please correct the typo lenght -> length or len.